### ISSUE TITLE
How to debug margin and scale?

### ISSUE THREAD

**Issue by AITTSMD** [happynear/AMSoftmax#23]
Hi, thanks for your great job. I wonder how I can debug the value of margin and scale to get better result.I use the default setting(m=0.35, scale=30)on my face recognition dataset, the final training loss  is about 3 and it can't decrease. So I come here to ask the quesion, thank you~

**Comment by happynear** [comment]
The final loss value in AM-Softmax is meaningless. It doesn't matter what the final loss is.

**Comment by AITTSMD** [comment]
@happynear okay，thank you~Can you explain the meaning of scale？How to adjust the value of scale to get better result?

**Comment by shiyuanyin** [comment]
@AITTSMD 
您好，
请问我想用 arcface 损失函数是将您的caffe 版本中的  将这一层， name: "label_specific_margin"
type: "LabelSpecificAdd"     type改动，LabelSpecificHardMargin,  不需要参数设置，
